src/components/ProfileForm.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormLabel from '@material-ui/core/FormLabel';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 300,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+export default function ProfileForm() {
+  const classes = useStyles();
+  const [grade, setGrade] = React.useState('');
+  const [gradYear, setGradYear] = React.useState('');
+  const [major, setMajor] = React.useState('');
+  const [gender, setGender] = React.useState('');
+  const [daca, setDaca] = React.useState('');
+
+  return (
+    <div>
```

Comment:
This should probably be a `<form>` and there should probably an `onSubmit` handler that gets triggered when the `Next` button is clicked.